### PR TITLE
Loadout Option Update: Attack of the Cones

### PR DESCRIPTION
**Pull request by EmilitiaEnnehrt**
## About The Pull Request
Apologies for the mess, first PR ever.

Adds more already-existing clothing from around the station as loadout option.

Also lowers or removes cost of some simple clothings with no special effect.

List of things added to the loadout option: waistcoat, maid apron, red armband, fake blindfold, circle glasses, bowler hat, witch hat, CONE HAT, italian moustache, gold necklace, beads necklace, zebra scarf, horrible tie, russian boots, fancy cowboy boots, yellow singer boots, overalls apron, henchman jumpsuit, overalls jumpsuit, corset, white suit skirt, amish suit, tuxedo, vice suit, waiter suit, black two piece suit, executive suit, female executive suit, black tango dress, western bustle dress, flamenco dress, sailor dress, yellow singer dress, red evening gown, draculass dress, and wedding dress.

## Why It's Good For The Game
People will be able to compose their outfit with more varied looks without needing to go through the troubles of buying them at the dorms.

## Changelog
:cl:
add: Added new options in the character creation loadout
balance: Changed price cost for some loadout clothing
/:cl:

### CODE REVIEW

File: code/modules/client/loadout/neck.dm [hunk 1]
```diff
@@ -5,6 +5,13 @@
 /datum/loadout_item/neck/headphones
 	path = /obj/item/clothing/ears/headphones
 
+/datum/loadout_item/neck/necklace
+	path = /obj/item/clothing/neck
```
Comment (Azarak) [inline code/modules/client/loadout/neck.dm:9]:
```suggestion
```
The `/obj/item/clothing/neck` is an abstract type

**Review by Azarak — COMMENTED** [review]
Just this change requested

**Comment by EmilitiaEnnehrt** [comment]
> Just this change requested

My bad, should be done now, I thought it was actually referencing necklace

File: code/modules/client/loadout/uniform.dm [hunk 2]
```diff
@@ -25,44 +31,43 @@
 /datum/loadout_item/uniform/kilt
 	path = /obj/item/clothing/under/costume/kilt
 
+/datum/loadout_item/uniform/psychedelic
+	path = /obj/item/clothing/under/misc/psyche
+
```
Comment (Azarak) [inline code/modules/client/loadout/uniform.dm:36]:
```suggestion
```
Really not a fan of this one, hurts to look at, people shouldnt be able to acquire in through loadout

**Review by Azarak — COMMENTED** [review]
This one I missed too

**Comment by EmilitiaEnnehrt** [comment]
> This one I missed too

Done, also strikes them off the PR's desc

**Review by Azarak — APPROVED** [review]

**Pull request merged**